frontend/src/hooks/queryBuilder/useCreateAlerts.tsx

```diff
@@ -17,7 +19,7 @@ import { Widgets } from 'types/api/dashboard/getAll';
 import { GlobalReducer } from 'types/reducer/globalTime';
 import { getGraphType } from 'utils/getGraphType';
 
-const useCreateAlerts = (widget?: Widgets): VoidFunction => {
+const useCreateAlerts = (widget?: Widgets, caller?: string): VoidFunction => {
```

Comment:
Why are we updating hooks for the sake of sending logEvents?  We should use whatever information we have and send events accordingly.

Reply:
Without this we can't differentiate on which screen the action was done. It's important to know screen of action.